### PR TITLE
GH-68: Add eslint and prettier to codefactor

### PR DESCRIPTION
**Pull request by jamestlee513**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "printWidth": 80,
+    "tabWidth": 2,
```
Comment (yauj) [inline .prettierrc:3]: 2 or 4?

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "printWidth": 80,
```
Comment (yauj) [inline .prettierrc:2]: Hmmm. 100? Scared that 80 isn' wide enough.

File: .prettierrc [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "tabWidth": 2,
+    "useTabs": false,
```
Comment (yauj) [inline .prettierrc:4]: Use tabs, not space. Technically saves space.

File: .prettierrc [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    "semi": true,
+    "singleQuote": true,
+    "quoteProps": "as-needed",
+    "jsxSingleQuote": false,
+    "trailingComma": "all",
+    "bracketSpacing": true,
+    "jsxBracketSameLine": true,
+    "arrowParens": "avoid",
```
Comment (yauj) [inline .prettierrc:12]: Can we do always? I personally prefer parenthesis, but we can fight on this one.

**Review by yauj — CHANGES_REQUESTED** [review]
Is there a way you can test this locally? And fix everything that needs to be fixed?

**Review by yauj — APPROVED** [review]

**Review by DBarnes2 — APPROVED** [review]

**Pull request merged**